coalib/output/ConsoleInteraction.py

```diff
@@ -250,7 +252,9 @@ def print_result(console_printer,
     console_printer.print(format_lines(result.message))
 
     if interactive:
-        cli_actions = CLI_ACTIONS
+        # FIXME: Handle duplicate actions of same type across
+        # NATIVE_ACTIONS and result.actions
+        cli_actions = NATIVE_ACTIONS + tuple(result.actions)
```

Comment:
btw the strategy I had in mind for dealing with it was to remove duplicate actions of same class type. In case of  duplicacy, the action having non-empty `next_actions` field is preferred. If the duplicity still presists,  the most recent action is given preference.